dblookupext/historyRepository.go

```diff
@@ -168,6 +170,30 @@ func (hr *historyRepository) RecordBlock(blockHeaderHash []byte,
 		}
 	}
 
+	for txHash, tx := range txResultsFromPool {
+		innerTxs := tx.GetUserTransactions()
+		if len(innerTxs) == 0 {
+			continue
+		}
+
+		for _, innerTx := range innerTxs {
```

Comment:
can you extract this into a method?

hr.recordInnerTx

Reply:
done